packages/components/src/Avatar/styles.tsx

```diff
@@ -3,6 +3,9 @@ import { Avatar } from '@mui/material';
 import { styled } from '../styles/styled';
 
 export const StyledAvatar = styled(Avatar)`
-  font-size: ${({ theme }) => theme.typography.h5.fontSize};
+  font-size: ${({ theme }) =>
+    theme.breakpoints.down('sm')
```

Comment:
Использована некорретная конструкция, на десктопе у тебя сейчас не верный размер шрифта
```suggestion
     ${({ theme }) => theme.breakpoints.down('sm')} {
    font-size: ${({ theme }) => theme.typography.h7.fontSize};
  }
```

Для стилей под другие разрешения в начале необходимо обернуть блок в медиа-выражение, и внтури него уже описывать стили.